### PR TITLE
ICMSLST-874 Add CFS schedule endpoints to add, edit and delete schedules.

### PR DESCRIPTION
**Pull request by MattHolmes123**

### CODE REVIEW

File: web/domains/case/export/views.py [hunk 1]
```diff
@@ -198,16 +204,73 @@ def submit_com(request: AuthenticatedHttpRequest, *, application_pk: int) -> Htt
...
+            "process": application,
+            "task": task,
+            "form": form,
+            "schedules": schedules,
+        }
+
+        return render(request, "web/domains/case/export/edit-cfs.html", context)
+
+
+@login_required
```
Comment (kevinetienne) [inline web/domains/case/export/views.py:238]: it looks like we need `@require_POST`?

File: web/templates/web/domains/case/export/edit-cfs.html [hunk 2]
```diff
@@ -30,11 +30,54 @@ <h4>Application details</h4>
...
+        </td>
+      </tr>
+    {% endfor %}
+    </tbody>
+  </table>
+  <hr>
+  <a href="{{ icms_url('export:cfs-schedule-add', kwargs={'application_pk': process.pk}) }}"
+     class="button small-button icon-plus">
+    Add Schedule
+  </a>
```
Comment (kevinetienne) [inline web/templates/web/domains/case/export/edit-cfs.html:82]: Should be wrapped in a post.

Comment (MattHolmes123) [reply web/templates/web/domains/case/export/edit-cfs.html:82]: yep agreed

File: web/domains/case/export/views.py (hunk 1 above)
Comment (MattHolmes123) [reply web/domains/case/export/views.py:238]: yep

**Review by kevinetienne — APPROVED** [review]

**Pull request merged**